prac_05/word_occurrences.py

```diff
@@ -0,0 +1,17 @@
+"""
+Word Occurrences
+Estimate: 10 minutes
+Actual: 22 minutes
+"""
+word_to_frequency = {}
+# text = input("Enter Text: ")
+text = "this is a collection of words of nice words this is a fun thing it is"
+words = text.split()
```

Comment:
Could this be combined with the for statement to create something like "for word in text.split()"?

Reply:
That's a good point.